python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -5,7 +5,7 @@
     import unittest
 
 
-def start_ganga():
+def start_ganga(gangadir_for_test='$HOME/gangadir_testing', extra_opts=[]):
```

Comment:
Could we document these new parameters, either here or in `GangaUnitTest.setUp` below?